CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ## [2.0.0-SNAPSHOT](https://github.com/cucumber/cucumber-jvm/compare/v1.2.5...master) (In Git)
 
+* [Java] Reduce Throwable to Exception in JavaSnippet ([#1207](https://github.com/cucumber/cucumber-jvm/issues/1207), [#1208](https://github.com/cucumber/cucumber-jvm/pull/1208) M.P. Korstanje)
```

Comment:
AFAIK, we discourage updating the Changelog in PR:s, and instead put that task on the person merging the PR.

Reply:
Planned on doing that myself. Then realized @mlvandijk could do it as a practice.

Reply:
But I was too slow ;)

Reply:
Unfortunately, the same day I was up to release Cucumber-JVM 2.0.0 was the wrong day for such practice ...

Reply:
Such is life. :D